.github/ISSUE_TEMPLATE/outdated-report.yml

```diff
@@ -0,0 +1,45 @@
+name: Package Outdated Report
+description: A package is outdated and no new version have been submitted for moderation.
+title: "(packageName) "
+labels: ["01 - _Triaging", "Outdated"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to report an outdated package. To validate the request please fill out the necessary items.
+
+        Make sure that the title of the issue starts with the package identifier that is outdated, inside paranthesis.
+  - type: checkboxes
+    attributes:
+      label: Checklist
+      description: Before continuing, make sure that you have done the following.
+      options:
+        - label: I have checked the moderation queue that no new version have been submitted (only visible when logged in), or the submitted package version have failed the Automated Checks.
```

Comment:
Can we add URL's here to the moderation queue (not sure if they can be added in these fields).

Reply:
I think it should be possible, I'll take a look to get it added.

Reply:
I have added a link to that section, I am not entirely fond of how it shows up due to the length of the checkbox label though.

![image](https://user-images.githubusercontent.com/1474648/149907759-79d33dde-65cf-4408-abab-6d4c2ae39582.png)